src/Rokt-Kit.js

```diff
@@ -99,10 +100,13 @@ var constructor = function () {
     }
 
     function selectPlacements(options) {
-        var placementAttributes = mergeObjects(
-            options?.attributes,
-            self.userAttributes
-        );
+
+        const optimizelyAttributes = self.onboardingExpProvider === 'Optimizely' ? fetchOptimizely() : {};
+        const placementAttributes = {
+            ...options?.attributes,
```

Comment:
I would advise against using the spread operator here. In my [other PR](https://github.com/mparticle-integrations/mparticle-javascript-integration-rokt/pull/9) I have an old school mergeObject function you can leverage.